plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -369,7 +369,7 @@ class CycloneDxReporter : Reporter {
             val outputFile = outputDir.resolve("$outputName.$fileExtension")
 
             val bomGenerator = when (fileExtension) {
```

Comment:
the variable name `bomGenerator` does not fit anymore. Maybe just name it `bom` ?